.github/workflows/ossf-scorecard-reporting.yml

```diff
@@ -17,15 +18,19 @@ jobs:
   security-scoring:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
Aside: Might make sense to port this over to keep this up-to-date https://github.com/nodejs/remark-preset-lint-node/blob/main/.github/dependabot.yml

Reply:
Also, considering we're moving to hash, might make sense to pin the version by hash. Pretty similar to:  https://github.com/nodejs/node/pull/46294

Reply:
Great catch @RafaelGSS, pinned versions in [743e582](https://github.com/nodejs/security-wg/pull/886/commits/743e582706ef7517a381f77d0d764c1cb69d3912)

@nschonni not sure I understood you point 🤔. Can you provide more details?

Reply:
Sorry, it wasn't a request that needs to be integrated here, but just flagging that this fell out of date because it isn't using Dependabot, which would have kept this up-to-date. I pointed to the other repo's config that could be possible leveraged, but I'm just going to separately PR that myself, as that shouldn't impact this PR

Reply:
Now is clear for me, thanks for the explanation @nschonni  😄

Reply:
No problem, I opened up https://github.com/nodejs/security-wg/pull/887 for the Dependabot stuff